messaginginapp/src/main/java/io/customer/messaginginapp/gist/data/model/Message.kt

```diff
@@ -13,7 +13,8 @@ data class GistProperties(
     val elementId: String?,
     val campaignId: String?,
     val position: MessagePosition,
-    val persistent: Boolean
+    val persistent: Boolean,
+    val overlayColor: String?
```

Comment:
also would be consistent to `MessagePosition`

Reply:
I kinda think color default values belong in the presentation logic not at the model so I prefer to have it here. Also defining this as non-null might be a bit misleading. But I see the point about consistency with that other field as well.

I can move it to the model if you feel strongly about it.

Reply:
> Also defining this as non-null might be a bit misleading

This is interesting, from the code it feels like we do indeed set an overlay with default color when user has not provided its override? if thats the case, then technically its not nullable? 

Either way, I don't feel strongly about it but I do think, the two things that make me think it make be more easier to have this at data layer other than the presentation
- We won't have to check for null values, every time. Because we are first going to get the message, check for nullable properties, and then check again for nullable overlay. 
- Consistency with other fields. 
 
 So a non-blocking opinion :)

Reply:
I see your point, I just think it's confusing for someone looking at the activity, if we set the default at data layer, they would think the value is coming from server while in fact it might not be.

They would have to know that this value has a hard-coded fallback at the parsing layer.

I vote we keep the default at the activity for clarity.